python/feathub/processors/flink/ast_evaluator/functions.py

```diff
@@ -16,7 +16,18 @@
 from feathub.common.utils import to_java_date_format
 
 
+"""
+This set contains the name of the built-in functions whose name
+is the same between Feathub and in Flink.
+"""
+_functions_with_equal_name = {"LOWER"}
+
+
 def evaluate_function(func_name: str, args: List[Any]) -> str:
-    if func_name.upper() == "UNIX_TIMESTAMP" and len(args) > 1:
-        args[1] = to_java_date_format(args[1])
-    return f"{func_name}({', '.join(args)})"
+    if func_name.upper() in _functions_with_equal_name:
```

Comment:
It might be simpler to consistently make func_name upper-case in `feathub.dsl.ast.FuncCallOp` than having each processor make this change when reading the function name.